packages/liveblocks-client/package.json

```diff
@@ -59,8 +46,8 @@
     "msw": "^0.39.1",
     "node-fetch": "2.6.7",
     "rollup": "^2.68.0",
+    "rollup-plugin-command": "^1.1.3",
     "rollup-plugin-dts": "^4.1.0",
-    "rollup-plugin-esbuild": "^4.8.2",
```

Comment:
Not really sure what we needed `esbuild` for here. I started trying to get this to work with the normal typescript plugin, and assumed we only needed to use `esbuild` for speed, but the build is as fast now as it is on `main`, so I figured if we don't need `esbuild`, having fewer build tool in the mix is better. (I think following the same reasoning, perhaps we can get rid of Babel here too and let the TypeScript plugin output those CJS.)

Please let me know if I'm missing the rationale for `esbuild` or whether I'm overlooking something 🙏

Reply:
We currently use `esbuild` because the last time I refactored our build system, I took inspiration from : https://github.com/pmndrs/zustand/blob/main/rollup.config.js

Performance is not an issue for our small bundle, so let's remove it if the only impact it does not impact the output